libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/internal/route/line/CacheResultUtilsTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.navigation.ui.maps.internal.route.line
+
+import com.mapbox.navigation.ui.maps.internal.route.line.CacheResultUtils.cacheResult
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class CacheResultUtilsTest {
```

Comment:
This test doesn't cover any overflow (and overwrite) functionality and whether the `maxSize` variable is respected.

Reply:
I added a test for the maxSize. I don't know what you mean by "overflow".